linuxkit.yml

```diff
@@ -6,6 +6,7 @@ init:
   - linuxkit/runc:d5cbeb95bdafedb82ad2cf11cff1a5da7fcae630
   - linuxkit/containerd:b6ffbb669248e3369081a6c4427026aa968a2385
   - linuxkit/ca-certificates:67acf038c44bb191ebb704ec7bb39a1524052cdf
+  - linuxkit/wireguard-tools:26fe3d38455f2d441549e3c54bdec1b26ac819b8
```

Comment:
I assume, though, that many people will use this to make transparent VPN proxies. That is -- boxes that have no ssh or serial or shells at all, but just configure a VPN and enable packet forwarding. Especially on Mac, people have expressed interest in this type of configuration. (And LinuxKit/Moby provides a clear way of doing that.)

So, with that in mind, I'm not sure it makes so much sense to bind this to sshd or getty. It belongs at the same level as `iproute2`. So, wherever that is is where wireguard-tools should go.